Lib/uuid.py

```diff
@@ -608,7 +608,7 @@ def _random_getnode():
     # significant bit of the first octet".  This works out to be the 41st bit
     # counting from 1 being the least significant bit, or 1<<40.
     #
-    # See https://en.wikipedia.org/wiki/MAC_address#Unicast_vs._multicast
+    # See https://en.wikipedia.org/wiki/MAC_address#Universal_vs._local_(U/L_bit)
```

Comment:
Seems like this should be https://en.wikipedia.org/wiki/MAC_address#Unicast_vs._multicast_(I/G_bit)